index.html

```diff
@@ -326,41 +357,49 @@ <h2 class="top-heading">About &nbsp; &nbsp; &nbsp; &nbsp; Myself</h2>
           <img class="sep1" src="./images/Separator Bottom.png" alt="line" />
         </div>
       </section>
+
       <section class="Contact-me">
-        <h2>Contact me</h2>
-        <p>
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it
-        </p>
-        <form action="https://formspree.io/f/{form_id}" methode="post">
-          <input
-            type="text"
-            name="user name"
-            placeholder="Girma Anwaar"
-            required
-            maxlength="30"
-          />
-          <input
-            type="email"
-            name="email"
-            placeholder="girma@gmail.com"
-            required
-            placeholder="user name"
-            required
-            maxlength="30"
-          />
-          <textarea
-            name="message"
-            id="message"
-            cols="30"
-            rows="10"
-            required
-            maxlength="500"
-            placeholder="Write your message here"
-          ></textarea>
-          <button type="submit">Get it Touch</button>
-        </form>
+        <div class="background-contact">
+          <h2>Contact me</h2>
+          <p>
+            If you have an application you are interested in <br />
+            developing, a feature that you need built or a project <br />
+            that needs coding. I’d love to help with it
+          </p>
+          <div class="form">
+            <form action="https://formspree.io/f/{form_id}" methode="post">
```

Comment:
The form looks pretty nice, but please make sure to increase its width like the one in the Figma template.

[Optional] It's highly recommended to use the correct `form_id` so that you can receive the user entry.

![image](https://user-images.githubusercontent.com/34779928/194423731-af349962-0cb4-4795-aa65-2c11dacff19c.png)